ex03.ex

```diff
@@ -101,9 +115,11 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
-
-
+def list_equal([],[]), do: []
```

Comment:
# -5

This just checks the lengths.

Why do 

~~~
def list_equal([h1 | t1], [h2 | t2]) do
   check_length(1 + length(t1), 1 + length(t2)) 
end
~~~

and not

~~~
def list_equal(list1, list2) do
   check_length(length(list1), length(list2)) 
end
~~~

(Of course, `length/1` is a library function, which you can't use.

I was looking for something like: 

~~~ elixir
  def list_equal([], []),                              do: true
  def list_equal([ h | t1 ], [h | t2]),                do: list_equal(t1, t2)
  def list_equal(a, b) when is_list(a) and is_list(b), do: false
~~~